x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/enrichments/__mocks__/alerts.ts

```diff
@@ -196,8 +202,14 @@ export const createAlert = (
       rule_name_override: undefined,
       timestamp_override: undefined,
     },
+    [LEGACY_ALERT_HOST_CRITICALITY]: undefined,
+    [LEGACY_ALERT_USER_CRITICALITY]: undefined,
     [ALERT_HOST_CRITICALITY]: undefined,
     [ALERT_USER_CRITICALITY]: undefined,
+    [ALERT_HOST_RISK_SCORE_CALCULATED_LEVEL]: undefined,
```

Comment:
What's the intent behind adding these to the mock alert, but making them undefined?

Reply:
It's for typescript, as those fields have 'string | undefined` type in the alert document.